platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/userlocation/MyLocationTrackingModeActivity.java

```diff
@@ -261,17 +284,33 @@ public void onLowMemory() {
     @Override
     public boolean onCreateOptionsMenu(Menu menu) {
         getMenuInflater().inflate(R.menu.menu_tracking, menu);
+        dismissLocationTrackingOnGestureItem = menu.findItem(R.id.action_toggle_dismissible_location);
+        dismissBearingTrackingOnGestureItem = menu.findItem(R.id.action_toggle_dismissible_bearing);
+        enableRotateGesturesItem = menu.findItem(R.id.action_toggle_rotate_gesture_enabled);
+        enableScrollGesturesItem = menu.findItem(R.id.action_toggle_scroll_gesture_enabled);
+        setCheckBoxes();
         return true;
     }
 
+    private void setCheckBoxes() {
+        if(mapboxMap != null && dismissBearingTrackingOnGestureItem != null) {
```

Comment:
Ok, fixing those. I notice there is a checkstyle.xml file in the platform/android directory, but it does not appear to match the style actually in use.